### PR TITLE
Unfinished solutions warning

### LINKED ISSUE #631: Unfinished solutions warning
When submitting section feedback whith unfinished solutions, warn user

### PR DESCRIPTION
**Pull request by nyet-ty**
Resolve #631

## PR includes

-   [x] Feature

### CODE REVIEW

**Comment by tsumo** [comment]
screenshot please

**Comment by nyet-ty** [comment]
image

**Comment by nyet-ty** [comment]
image

**Comment by nyet-ty** [comment]
image

File: src/components/SectionFeedback/SectionFeedback.tsx [hunk 1]
```diff
@@ -135,10 +135,29 @@ export const SectionFeedback = ({ section, isEditable, hasTasks }: SectionFeedba
                 });
             });
     });
+    const solutionsWithoutResultWarn = section.solutions.reduce((acc: string[], rec) => {
+        return rec.result === 'UNKNOWN' ? [...acc, rec.problem.name] : acc;
```
Comment (9teen90nine) [inline src/components/SectionFeedback/SectionFeedback.tsx:139]: acc.push is much faster

**Review by 9teen90nine — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

**Pull request merged**